### ISSUE TITLE
Non-interactive version?

### ISSUE THREAD

**Issue by indigoviolet** [redraw/gh-install#5]
Would be nice to have a non-interactive version for shell scripts

**Comment by iainelder** [comment]
By default:

* choose the latest release
* extract all executable files
* install to `$HOME/bin`

**Comment by redraw** [comment]
Hey! yeah, that would be nice, I'll take a look 

@iainelder currently it saves to `~/.local/bin`, but it can be overriden by setting the `GH_BINPATH` env.
Might also think about moving all executable files.

**Comment by iainelder** [comment]
That install location is fine and even better that there is an environment variable to control it.

I've tried a few other install-from-Github tools such as [Rishang's install-release](https://github.com/Rishang/install-release) and [Jooola's gh-release-install](https://github.com/jooola/gh-release-install). Each has its pros and cons.

I'd be happy to try what you come up with if you want to give us another option :-)

**Comment by redraw** [comment]
Meanwhile, you could give a try to [binenv](https://github.com/devops-works/binenv), looks promising and more complete. It knows how to install a lot of popular projects, and I guess you can extend the `$HOME/.config/binenv/distributions.yaml` file if you need another one

**Comment by iainelder** [comment]
Nice, didn't know about binenv! This looks like a more general tool as well as mainly supporting binaries from Github Releases.

Reminds me of [deb-get](https://github.com/wimpysworld/deb-get) for Apt packages.

**Comment by redraw** [comment]
yeah, I actually liked the simplicity of the current version, but I'm open to merge yours! just testing a few cases now
